devops/setup-gpg/action.yaml

```diff
@@ -0,0 +1,48 @@
+name: 'Setup GPG'
+description: 'Configures this action to run gpg with a given key and pass'
+inputs:
+  private-key:  # id of input
+    description: 'gpg private key'
+    required: true
+  key-pass:  # id of input
+    description: 'gpg key pass'
+    required: true
+  key-name:  # id of input
+    description: 'gpg key name'
+    required: true
+    default: "Aerospike"
+runs:
+  using: "composite"
+  steps:
+      - name: Set up GPG
+        env:
+          GPG_PRIVATE_KEY: ${{ inputs.private-key }}
+          GPG_KEY_PASS: ${{ inputs.key-pass }}
+          GPG_ID: ${{ inputs.key-name }}
```

Comment:
```suggestion
```
In your code inside `run:`, you can just refer to the inputs directly instead of defining environment variables. I believe Github Actions will censor the secrets in the logs automatically, even if they're defined as inputs in this action